client/modules/IDE/components/NewFileForm.stories.jsx

```diff
@@ -0,0 +1,8 @@
+import NewFileForm from './NewFileForm';
+
+export default {
+  title: 'IDE/NewFileForm',
+  component: NewFileForm
+};
+
+export const Default = {};
```

Comment:
The `NewFileForm` is only used inside of the `NewFileModal` so it might be redundant to have stories for both. (the same for the `NerFolderForm`)

TBH we could consider combing the two component files as there's not much left in the `NewFileModal` after we made a common `Modal` component.

Reply:
I like the idea of having fewer Modals and reusing the generic one. I think thats capturing the layout primitives

Reply:
I've removed `NerFolderForm`